cnd/build.rs

```diff
@@ -0,0 +1,11 @@
+use std::process::Command;
+fn main() {
+    let output = Command::new("git")
+        .args(&["rev-parse", "HEAD"])
+        .output()
+        .expect("failed to get commit hash for tip of branch");
```

Comment:
As mentioned during the call today: maybe it's nice to just print a warning instead of panicing. 

This is not a blocker, so feel free to just move forward with this.

Reply:
In a build script, I'd say it is actually better to panic otherwise the application will panic at runtime (env variable not being set).

Alternatively to `expect`, you could use `?` and have the `main` return an `anyhow::Result` but for a simple build-script like this, it doesn't really make a difference I believe :)